### PR TITLE
RHMAP-9815 send permission level to millicore if possible

### PR DESCRIPTION
**Pull request by pb82**
# Motivation

Append the `requestedAct` query parameter to the authorization request to millicore. This is used for a more fine grained (read/write level) control of authorization.

ping @ziccardi @wei-lee

### CODE REVIEW

**Comment by wei-lee** [comment]
LGTM

File: lib/mbaas.js [hunk 1]
```diff
@@ -30,6 +30,21 @@ function applyAuth(req, res,api,role,callback){
     }],callback);
 }
 
+/**
+ * DB actions are bound to specific permissions. For example the `list` action
+ * required only read permissions while `update` requires write. Those permissions
```
Comment (ziccardi) [inline lib/mbaas.js]: Small typo. `required` should be `requires`

**Comment by ziccardi** [comment]
LGTM. Just a typo in a comment.

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

**Comment by wei-lee** [comment]
published to npm

**Pull request merged**